### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by dmitry-ivanov**
Just a small typo found.

(And also experimenting with the Draft Pull Requests)

-----
[View rendered guides/onboarding/onboarding--mentorGuide.md](https://github.com/InteractionDesignFoundation/handbook/blob/mentor-guide-typo/guides/onboarding/onboarding--mentorGuide.md)

### CODE REVIEW

**Comment by dmitry-ivanov** [comment]
I think that Draft PRs is a great replacement for those [WIP] prefixes that we use.

They can't be merged as we can see, we have that "Draft" gray label on that page (see above, under the PR title), and also, on the PRs list, the icon is gray (which is useful too).

Anyways, waiting for Ekin's review of this feature too, to make sure that it fits all of our conventions :)

**Comment by gzg** [comment]
In my opinion, it looks great as a replacement for WIP. The only downside I could see is that WIP PRs are very catchy on the PR list. However, I think our eyes will get used to picking up the gray color in no time as well. :-) So it's still easy to spot draft PRs from the list. 👍 

On top of that, it's good that draft PRs cannot be merged. That's added benefit. 👍 

And well, it's native to GitHub; so that's perfect. :-)

In summary, I believe draft PRs can replace WIP. It looks like a better alternative.
